javascript.js

```diff
@@ -0,0 +1,56 @@
+function Murphy() {
+    alert("Murphy, Hound Dog, $150.89");
+  }
+
+ function Poppy() {
+    alert("Poppy, Floof, $123.45");
+  }
+
+ function Jack() {
+    alert("Jack, Beagle, $100.00")
+}
+
+ function Duffy() {
+    alert("Duffy, Pit Bull, $123.45")
+}
+
+ function Lucas() {
+    alert("Lucas, Austrian Hound, $123.45")
+}
+
+ function Jake() {
+    alert("Jake, Yellow Lab, $123.45")
+}
+
+ function Angus() {
+    alert("Angus, Bull Mastiff, $123.45")
+}
+
+ function Violet() {
+    alert("Violet, Chocolate Lab, $123.45")
+}
+
+ function Piper() {
```

Comment:
Great! But all these functions are doing the same thing. A better way is to do it is pass the parametes in via your click handler and use the same function to do your work.  See an example at https://codepen.io/jimmynono/pen/RmbzQb?editors=1010